src/styles/blocks/card.scss

```diff
@@ -0,0 +1,99 @@
+@import "../colors";
+
+.card {
+  font-family: Roboto, serif;
+  width: 200px;
+  height: 408px;
```

Comment:
- Don't fix container size (if there is no such a requirement), height in your case
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/0ff02c7b-5b8a-4da0-a7aa-9aae6091eda5)